streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -95,6 +98,28 @@ public void cleanup() {
         }
     }
 
+    @Test
+    public void testOsDefaultSocketBufferSizes() {
```

Comment:
nit: naming -> `shouldAcceptDefaultBufferSizes()`

Also, I am wondering why we check for default buffer size? The ticket was about the issue, that `-1` was not accepted. Thus, while having this test is ok, we should actually test for `-1` to have a test that covers the reported issue.